Client/mods/deathmatch/logic/rpc/CElementRPCs.cpp

```diff
@@ -48,6 +48,7 @@ void CElementRPCs::LoadFunctions()
     AddHandler(SET_WEAPON_OWNER, SetWeaponOwner, "setWeaponOwner");
     AddHandler(SET_CUSTOM_WEAPON_FLAGS, SetWeaponConfig, "setWeaponFlags");
     AddHandler(SET_PROPAGATE_CALLS_ENABLED, SetCallPropagationEnabled, "setCallPropagationEnabled");
+    AddHandler(COLSHAPE_POLYGON_SET_HEIGHT, SetColPolygonHeight, "setColShapePolygonHeight");
```

Comment:
Perhaps rename to `SET_COLPOLYGON_HEIGHT` for consistency.
```suggestion
    AddHandler(SET_COLPOLYGON_HEIGHT, SetColPolygonHeight, "setColShapePolygonHeight");
```